e2e/README.md

```diff
@@ -1,41 +1,124 @@
+- Install from Android Studio [these instructions](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#installing-from-android-studio)
 
-## Android
+2.(Optional) [Set up quick boot](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#emulator-quick-boot) on the emulator.
 
-### Initial setup
+**NOTE**: The AOSP emulators appear to require Java 8 (JDK 1.8). You may have a newer version of Java, like Java 15 installed...
 
-1. Get yourself one or more emulators, you have a couple options:
+**IMPORTANT JAVA NOTICE**
 
-- [Detox recommends Android Open-Source Project (AOSP) emulators](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#android-aosp-emulators)
-- Install from Android Studio [these instructions](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#installing-from-android-studio)
+Read instructions on how to install older versions of Java alongside modern Java versions, and easily switch between them: [-> java.md <-](java.md)
+
+## Tests
+
+### Onboarding
+A common tests to get through the onboarding. Often used before other test to get deeper into the app.
+
+### ExploreDemoMenu
+
+e2e testing Detox script that iterates through every page of the app available on the debug menu and takes screenshots as it goes.
+
+See the `artifacts` folder for the output.
+
+## Running Tests
+
+### Preparation
+
+`yarn pre:test:android`
+
+or
+
+`yarn pre:test:ios`
+
+### Metro server
+
+If you haven't recently run a `yarn pre:test:ios` command, or if you've run it but closed the many   prompts it opened, you might not have metro server running. *Metro must be running* for Detox to talk with React and to interact with the app.
+
+From a separate terminal tab, or prompt, navigate to the app repo and run `npx react-native start`
+
+![Setup to run Detox](SetupToRunDetox.png)
+
+**BUILDING THE BUNDLE TAKES A WHILE**
+See progress bar on bottom of Metro server window.
+
+### Individual tests
+
+*As outilned in `.detoxrc.json`*
 
-2. (Optional) [Set up quick boot](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#emulator-quick-boot) on the emulator.
+From the main repo, for example, run:
 
-**NOTE**: The AOSP emulators appear to require Java 8. You may have a newer version of Java like Java 14 installed. Here are some instructions on [how to install older versions of Java and switch them](java.md)
+`detox test e2e/exploreDemoMenu.e2e.js --configuration=ios`
 
-### Running the tests
+or `android.aosp` for android simulator
 
-1. `yarn pre:test:android`
-2. `yarn test:android`
+![Succesful Detox run of exploreDemoMenu.e2e.js](SuccesfulDemoMenuTest.png)
+Took 100s on a 2017 13-inch Macbook Pro.
+
+### Run the entire test suite
+
+This takes many minutes, and binds up most of your computer ressources while at it...
```

Comment:
Could say `this may take several minutes`

Reply:
Updated in lastest commit